### PR TITLE
#159951814 Setup react router version 4 (React-router-dom)

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### What does this PR do?
This PR adds routing to the application using react router and react router dom.
#### Description of Task 
Have the following routes working 
    * "/"
    * "/login"
#### How should this be manually tested?
After cloning the application, cd into ah-frontend-taces and click the links "Home" or "Login"
#### Any background context you want to provide?
The react router dom version used is v4.3.1
#### What are the relevant pivotal tracker stories?
[#159951814](https://www.pivotaltracker.com/story/show/159951814)

### CODE REVIEW

File: src/components/Home/Home.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.Home{
+    color : aqua;
+    
+}
```
Comment (abulojoshua1) [inline src/components/Home/Home.css]: Please put an End of File here.

Comment (AGMETEOR) [reply src/components/Home/Home.css]: Thank you. Let me work on it.

**Review by phillipseryazi — APPROVED** [review]

File: src/routes.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import { Route, Switch, BrowserRouter } from "react-router-dom";
+import Home from "./containers/Home/Home";
+import Login from "./containers/Login/Login";
+import Wrapper from "./hoc/Wrapper/Wrapper";
+import Header from "./components/Header/Header";
+
+const routes = () => {
+  return (
+    <BrowserRouter>
+    <Wrapper>
```
Comment (geofrocker) [inline src/routes.js:11]: what happens when you remove this line

Comment (AGMETEOR) [reply src/routes.js:11]: Wrapper is an HOC that allows Switch and Header component to sit side by side.

Comment (AGMETEOR) [reply src/routes.js:11]: screen shot 2018-08-28 at 14 38 40

Comment (geofrocker) [reply src/routes.js:11]: Okay

**Review by ssewilliam — APPROVED** [review]

**Review by abulojoshua1 — APPROVED** [review]

**Review by lytes20 — APPROVED** [review]

**Review by RutaleIvanPaul — APPROVED** [review]

**Pull request merged**